### PR TITLE
added a new quote

### PR DESCRIPTION
**Pull request by Khush-Ramdev**
## ✅ New Quote Checklist
- [ x ] The quote(s) added are in alphabetical order and not ❌ duplicates.
- [ x ] The quote(s) added are about **learning a foreign language**.
- [ x ] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [ x ] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -23,6 +23,10 @@
     "author": "Amy Chua",
     "sentence": "Do you know what a foreign accent is? It’s a sign of bravery."
   },
+  {
+    "author": "Amy Santiago",
+    "sentence":"Some Say latin is a dead language but I say its just taking a long nap."
```
Comment (Kvaibhav01) [inline server/data/quotes.json:28]: Thanks for the PR. Please update it as shown:

```suggestion
    "sentence": "Some Say latin is a dead language but I say its just taking a long nap." 
```

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]

File: server/data/quotes.json [hunk 2]
```diff
@@ -23,6 +23,10 @@
     "author": "Amy Chua",
     "sentence": "Do you know what a foreign accent is? It’s a sign of bravery."
   },
+  {
+    "author": "Amy Santiago",
+    "sentence": "Some Say latin is a dead language but I say its just taking a long nap."
```
Comment (jayehernandez) [inline server/data/quotes.json:28]: Please add these improvements

```suggestion
    "sentence": "Some say Latin is a dead language but I say it's just taking a long nap." 
```

File: server/data/quotes.json [hunk 3]
```diff
@@ -23,6 +23,10 @@
     "author": "Amy Chua",
     "sentence": "Do you know what a foreign accent is? It’s a sign of bravery."
   },
+  {
+    "author": "Amy Santiago",
```
Comment (jayehernandez) [inline server/data/quotes.json:27]:
```suggestion
    "author": "Amy Santiago in Brooklyn Nine Nine",
```

**Review by jayehernandez — CHANGES_REQUESTED** [review]

**Review by jayehernandez — APPROVED** [review]

**Comment by jayehernandez** [comment]
Thank you for contributing to Letra ☀️, @Khush-Ramdev!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**